modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -154,6 +154,10 @@ echo "Forseti Startup - Enabling and starting Forseti service."
 systemctl enable --now forseti
 echo "Forseti Startup - Success! The Forseti API server has been enabled and started."
 
+# Increase Open File Limit
+echo "ubuntu soft nofile 32768" | sudo tee -a /etc/security/limits.conf
```

Comment:
I think it would be good to have this limit as a Terraform variable. It's possible that some users will want/need to set this to a different value and will be easier to maintain/change this in the future as a variable.

Reply:
That is a nice to have feature. Setting it to 32768 fixes the `open files error`. We can open an issue to make ulimit modifiable for later.

Reply:
I agree that it's a great idea to be in terraform, but for simplicity purpose, would like to have a real use case before adding more to terraform.  So, okay to add this here for now.

Reply:
Opened [an issue](https://github.com/forseti-security/terraform-google-forseti/issues/429) to track this and make ulimit configurable.